### ISSUE TITLE
About Gram anchoring

### ISSUE THREAD

**Issue by Chensihao-7** [marjanstoimchev/DinoV3LightningTraining#3]
Thank you for your excellent work. I have a question about the Gram anchor in step 2: which checkpoint from step 1 should I use as input? In step 1, I also prepared my own validation set to compute the validation loss—would it be better to use the checkpoint with the lowest validation loss? Thank you for your help.
